ui/cap-react/src/components/drafts/form/themes/grommet/templates/FieldTemplate.js

```diff
@@ -62,7 +78,12 @@ let FieldTemplate = function(props) {
         <span>
           <span
             style={{ color: "#000" }}
-            dangerouslySetInnerHTML={{ __html: label }}
+            dangerouslySetInnerHTML={{
+              __html:
+                uiSchema["ui:options"] && uiSchema["ui:options"].titleIsMarkdown
+                  ? sanitizedMarkedInput(markedInput(label))
+                  : label
```

Comment:
we shouldn't pass `label` in `setInnerHTML` as dangerous. we should either sanitise or pass the normal way. same in other cases